acceptance-tests/apps/postgresqlapp/go.mod

```diff
@@ -1,6 +1,6 @@
 module postgresqlapp
 
-go 1.19
+go 1.18
```

Comment:
Is there any reason why we are downgrading go version in this app?

Reply:
That was introduced temporarily, for the PAS-shipped buildpack, will revert.